doc/source/user/configoptions.rst

```diff
@@ -411,6 +411,33 @@ General options
 
       Controls whether curved geometries should be approximated by linear geometries.
 
+- .. config:: OGR_ORGANIZE_POLYGONS
+     :choices: DEFAULT, SKIP, ONLY_CCW, CCW_INNER_JUST_AFTER_CW_OUTER
+
+     Defines the method used to classify polygons as holes or shells. Although
+     one of the options is named ``DEFAULT``, some drivers may default to a
+     different method to reduce processing by taking advantage of a format's
+     constraints. The following methods are available, in order of decreasing
+     expected runtime:
+
+     - ``DEFAULT``: perform a full analysis of the topological relationships
+       between all polygons, classifying them as shells or holes and associating
+       them according to the OGC Simple Features convention.
+
+     - ``ONLY_CCW``: only attempt to classify polygons with a counter-clockwise
+       orientation, assuming that all polygons with a clockwise orientation
+       represent holes. This is the convention of the Shapefile format, whose
+       driver defaults to this method.
+
```

Comment:
This is **not** the Shapefile convention, the shapefile convention is exactly reversed: Clockwise rings are the outer rings, counterclockwise rings are holes. If the shapefile driver is using this method then either the name "ONLY_CCW" or the description is misleading.

Reply:
Jukka is right. I read the proposed doc addition too fast. The Doxygen of the function mentions "or only make it analyze counterclockwise polygons by adding METHOD=ONLY_CCW to the option list if you can assume that the outline of holes is counterclockwise defined (this is the convention for example in shapefiles, Personal Geodatabases or File Geodatabases)."

I realize this might not be crystal clear. Let's try to explain it better. In METHOD=ONLY_CCW mode, rings with clockwise orientation are assumed to be outer rings, and those with counterclockwise orientation to be inner rings. METHOD=ONLY_CCW  means that a topological analysis is only needed for those counterclockwise oriented rings to figure out to which outer ring they belong too